.eslintrc.js

```diff
@@ -50,197 +34,204 @@ const defaultRules = {
   }],
   'space-in-parens': [2, 'never'],
   'spaced-comment': [2, 'always'],
-  strict: 0,
+  strict: [2, 'never'],
```

Comment:
What is the reasoning behind adding this rule? Do we actually want to use [sloppy mode](https://developer.mozilla.org/en-US/docs/Glossary/Sloppy_mode)? (I guess not?) Or do we address the issues in a different way?

Reply:
So ES6 modules are always in strict mode automatically, so the declaration is redundant. That said, we have heaps  of old Angular code that would prolly benefit from having it, but in my understanding, we'll be safe just by nature of us modifying and adding mostly new  ES6 code and _removing_ old Angular "ES5" code.

Reply:
I see. Makes sense.
We could still benefit from strict mode in api code and tests, couldn't we? AFAIK it uses `require` and `module.exports` for exporting and importing. Have we used strict mode until now?